File: plugin-pac/src/java/fr/paris/lutece/plugins/pac/service/artifact/IArtifactService.java
package fr.paris.lutece.plugins.pac.service.artifact;

import org.springframework.transaction.annotation.Transactional;

import fr.paris.lutece.plugins.pac.bean.artifact.Artifact;
import fr.paris.lutece.plugins.pac.service.IPacService;

@Transactional
public interface IArtifactService extends IPacService<Integer, Artifact>
{

}

File: trunk/plugin-scala/src/main/scala/fr/paris/lutece/plugins/scala/service/artifact/IArtifactService.scala
package fr.paris.lutece.plugins.scala.service.artifact

import fr.paris.lutece.plugins.genericjpa.service.IPluginService
import fr.paris.lutece.plugins.genericjpa.service.AbstractService
import fr.paris.lutece.plugins.scala.bean.artifact.Artifact

trait IArtifactService extends IPluginService[Int,Artifact] {
}